src/libstd/sys_common/backtrace.rs

```diff
@@ -107,47 +107,66 @@ fn filter_frames(frames: &[Frame],
         "_ZN3std3sys3imp9backtrace",
         "ZN3std3sys3imp9backtrace",
         "std::sys::imp::backtrace",
+
         "_ZN3std10sys_common9backtrace",
         "ZN3std10sys_common9backtrace",
         "std::sys_common::backtrace",
+
         "_ZN3std9panicking",
         "ZN3std9panicking",
         "std::panicking",
+
         "_ZN4core9panicking",
         "ZN4core9panicking",
         "core::panicking",
+
         "_ZN4core6result13unwrap_failed",
         "ZN4core6result13unwrap_failed",
         "core::result::unwrap_failed",
+
         "rust_begin_unwind",
         "_ZN4drop",
```

Comment:
Er was this added in the previous PR? This definitely seems like something that we shouldn't be dropping, this is just a normal function?